gatorgrade/__init__.py

```diff
@@ -0,0 +1 @@
+"""Standard init file."""
```

Comment:
This should feature a docstring for the `gatorgrade` package. If the documentation for this package is not ready, I would at least suggest adding a stub like `"""GatorGrade is a program to do X."""`.